frontend/src/helpers/utils.js

```diff
@@ -58,6 +58,10 @@ export const formatDateToMonthDayYear = (date) => {
     return new Date(date).toLocaleString("en-US", { month: "long", day: "numeric", year: "numeric" });
 };
 
+// Is there a better place for this?
+// List of BLI statuses which are considered Draft
+export const draftBudgetLineStatuses = ["DRAFT", "UNDER_REVIEW"];
+
```

Comment:
🤔 was wondering about this, when the toggle is inactive, the "UNDER_REVIEW" BLIs don't show.